app/liquid_tags/unified_embed/tag.rb

```diff
@@ -27,6 +27,10 @@ def self.new(tag_name, input, parse_context)
       # Extract just the URL from the input, without any params, for validation
       actual_link = extract_only_url(stripped_input)
 
+      # When Listings are disabled, it makes little sense to perform a validate_link
+      # network call.
```

Comment:
Do you know if there's a way to put this in the Listings Tag (https://github.com/forem/forem/blob/mail/app/liquid_tags/listing_tag.rb)?

Reply:
Yup, fair question @jeremyf. That was definitely my original attempt. However, before we even get to ListingTag, we first have to get through `tag.rb` (the unified_embed version) and then ForemTag.

In `tag.rb`, the listing link will fail the `validate_link!` method with a 404 error, because it literally cannot be found.

I certainly welcome any improved approaches. While I planned my approach, I was sensitive to not over-engineering the solution. 😅

Reply:
Sounds great, and we can certainly proceed.